### ISSUE TITLE
All features of the exisiting SN?

### ISSUE THREAD

**Issue by bowstones** [ScratchNetworkLite/ScratchNetworkLite#1]
Will this have all of the same features as the current non-free ScratchNetwork? Or is it just a stripped-down version?

**Comment by ed6767** [comment]
Yes except for some features like PushPoints so it's more focused on community no paying to win.

**Comment by moeenio** [comment]
Yes, we need another system for pushing posts.

**Comment by ed6767** [comment]
They will be sorted by date modified

**Comment by moeenio** [comment]
Yes, and you can add a button to change how they're sorted.

**Comment by locness3** [comment]
BTW can I do a design ?
